testsuite/client-secret-jwt-keycloak-test/src/test/java/io/strimzi/testsuite/oauth/KeycloakClientCredentialsWithJwtValidationTest.java

```diff
@@ -30,11 +30,11 @@
 public class KeycloakClientCredentialsWithJwtValidationTest {
 
     private static final String HOST = "keycloak";
-    private static final String REALM = "demo";
+    private static final String REALM = "demo-ec";
```

Comment:
Don't we need to be able to test both EC and non-EC?

Reply:
The default used is RSA which is used in all the other tests. But this specific test I adjusted to use EC. It may be a bit obfuscated, but both the default and EC get tested over the course of a testsuite run. These tests take some time and some hardware resources to run, so I try to avoid doing permutations.

Reply:
OK, so long as we haven't lost all coverage of RSA that's fine.